Source/Frontend/UI/Extensions/NumericUpDownHexFix.cs

```diff
@@ -0,0 +1,117 @@
+namespace RTCV.UI
```

Comment:
See other comment on UI/Components/Controls. It may be good to do these one by one as there'll most likely be a _lot_ of changes for some of them.